torch/utils/data/datapipes/iter/web.py

```diff
@@ -0,0 +1,34 @@
+from io import IOBase
+from typing import Tuple
+from urllib.error import HTTPError, URLError
+import urllib.request as urllib
+from torch.utils.data import IterDataPipe
+
+
+class WebIterDataPipe(IterDataPipe[Tuple[str, IOBase]]):
+    r""" :class:`WebIterDataPipe`
+
+    Iterable DataPipe to load file url(s) (web url(s) pointing to file(s)),
+    yield file url and IO stream in a tuple
+    args:
+        timeout : timeout for web request
+    """
+
+    def __init__(self, source_datapipe, timeout=30):
```

Comment:
Can `timeout` be `None` as default to mimic the default behavior of `urllib`?

Reply:
fyi, yes agree. It is changed to None.